src/read-only-handler.ts

```diff
@@ -70,13 +75,17 @@ export class ReadOnlyHandler {
     }
     getOwnPropertyDescriptor(shadowTarget: ReactiveMembraneShadowTarget, key: PropertyKey): PropertyDescriptor | undefined {
         const { originalTarget, membrane } = this;
-        const { valueObserved } = membrane;
+        const { valueObserved, magicPropertyKey } = membrane;
 
```

Comment:
Add a test for the `o[flag]` is `undefined` result and then one for getting the descriptor.

Reply:
yeah... tests are coming...

Reply:
tests are in...